app.yaml

```diff
@@ -136,7 +136,7 @@ libraries:
 # when set to true. It allows minified resources to be used when
 # running the server in the dev environment.
```

Comment:
This environmental variable reflects if we are running the server in the dev environment.

Reply:
Done